apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V34__preadj_rda_replay_table_updates.sql

```diff
@@ -0,0 +1,28 @@
+/*
+ * Every claim change from the RDA API contains a 64-bit sequence number that can be used
+ * to request a replay of past updates.  Each claim in the database will have the most recent
+ * sequence number that affected that claim stored in a column.
+ */
+
+/*
+ * Add the sequence number column used to track which update from RDA API was
+ * used to create or update the claim.
+ */
+alter table "pre_adj"."FissClaims"
```

Comment:
That's a cool observation and not one that I was aware of.  It may not be necessary since postgresql 11 though  according to this blog:  https://www.depesz.com/2018/04/04/waiting-for-postgresql-11-fast-alter-table-add-column-with-a-non-null-default/

Reply:
Just to confirm the post is correct I tried out the SQL in the blog and the add column was instantaneous with 10,000,000 records.